requirement/svg/arrow-down.svg

```diff
@@ -1,5 +1,5 @@
 <!-- Generated by IcoMoon.io -->
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" class="new-shape" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
in the `readme.md` file there is a requirement, to **only** make changes in the `web` folder